libairspy/src/iqconverter_float.c

```diff
@@ -122,22 +137,22 @@ static _inline float process_fir_taps(const float *kernel, const float *queue, i
 
 #ifdef USE_SSE2
 
-		for (i = 0; i < it; i++)

```

Comment:
Please do a rebase on your code in order to avoid changing code not modified ... like this loop ...

Reply:
You should defined #define _inline to what is supported by FreeBSD like it is done for other Platform/Compiler ...